README.md

```diff
@@ -0,0 +1,34 @@
+# ibmcloud-image-builder
+
+
+This repo is for the open source project that build various virtual machine images in `qcow2` format that can be imported IBMCLOUD Cloud Object Storage (COS) and be served as custom images.
```

Comment:
You don't need to say that the project is open source.  That's obvious because the code is included.

Reply:
> You don't need to say that the project is open source. That's obvious because the code is included.

Thanks, I will update that at the next PR.